src/components/Modal/styles.module.scss

```diff
@@ -1,5 +1,5 @@
 .donationsContainer {
-  height: 500px;
+  height: auto;
```

Comment:
do we need this here?

Reply:
imo we can utilize flexbox to standardize container behavior